cds-plugin.js

```diff
@@ -297,7 +296,7 @@ module.exports = class AdvancedEventMesh extends cds.MessagingService {
       const msg = normalizeIncomingMessage(message.getBinaryAttachment())
       msg.event = event
       try {
-        await this.tx({ user: cds.User.privileged }, tx => tx.emit(msg))
+        await this.processInboundMsg({ user: cds.User.privileged }, msg)
```

Comment:
@David-Kunz was this function introduced with 8.9.1? could we so something like `this.processInboundMsg ? this.processInboundMsg() : this.tx()` to be backwards compatible?

Reply:
almost, we can do

```js
if (this.processInboundMsg) {
  await this.processInboundMsg({ user: cds.User.privileged }, msg)
} else {
  await this.tx({ user: cds.User.privileged }, tx => tx.emit(msg))
}
```

Reply:
what about the `inbound: true` change?